### PR TITLE
Change description of role in RequireRole method

### PR DESCRIPTION
**Pull request by woeterman94**
The description of the pram roles feels ambiguous. Make it clear that the user should belong to one of the roles and not all of them.
As documented: https://docs.microsoft.com/en-us/aspnet/core/security/authorization/roles?view=aspnetcore-2.1

### CODE REVIEW

**Comment by Eilon** [comment]
@blowdart can you share your thoughts on this new phrasing?

I agree the old phrasing was misleading. I'm not sure I'm crazy about the new wording either, but I do think it's better.

**Comment by blowdart** [comment]
It's better. How about "The collection/array of roles where membership of at least one of the roles fulfills the requirement"? Or is that too wordy?

**Comment by woeterman94** [comment]
That's too wordy, and perhaps more confusing.

**Comment by Eilon** [comment]
Discussed in triage - this change is better than what was previously there. Thanks for the PR!
